source/index.html.md

```diff
@@ -494,6 +497,13 @@ Exactly one payment method must be used.
   <dd>20[0-9]{2} <br /> Expiry year of card to charge.</dd>
   <dt>card[csc]</dt>
   <dd>[0-9]{3} <br /> <i>Optional when partner is trusted.</i> <br /> Card Security Code.</dd>
+  <dt>card[name]</dt>
+  <dd>
+    [\x20-\x7E]{1,35}
+    <i><a target="_blank" href="http://en.wikipedia.org/wiki/ASCII#ASCII_printable_characters">ASCII printable characters</a></i> <br />
+    <i>Optional</i> <br />
+    Name on card.
+  </dd>
```

Comment:
Do we want to make it available for authorizations?

Reply:
I lean towards "no".

`s/(Name on card.)/\1 Only relevant for credits./` - WDYT?

Reply:
Then I suspect the lines are just added the wrong place — to appear on https://docs.gateway.clearhaus.com/#authorizations when it should appear on https://docs.gateway.clearhaus.com/#credits

Reply:
Aah! Right! :man_facepalming: